internal/characters/amber/attack.go

```diff
@@ -43,7 +43,13 @@ func (c *char) Attack(p map[string]int) action.ActionInfo {
 	}
 	c.Core.QueueAttack(
 		ai,
-		combat.NewCircleHit(c.Core.Combat.PrimaryTarget(), 0.5),
+		combat.NewBoxHit(
+			c.Core.Combat.Player(),
+			c.Core.Combat.PrimaryTarget(),
+			combat.Point{Y: -0.5},
+			0.1,
+			1,
+		),
```

Comment:
comment in regard to all bow attacks that do not have an AoE:

Should we use `NewSingleTargetHit` instead? My concern with current hitbox implementation (and old) is a non aoe attack could hit multiple targets. My assumption with game mechanics is when arrow travels with above hitbox, first collision consumes (so can only have 1 collision). We do not have such logic in current implementation.

Reply:
This hitbox is centered on PrimaryTarget, so unless you have targets very close together this won't be real issue.